source/admin_manual/list_of_events.rst

```diff
@@ -1242,13 +1242,20 @@ Finished the message submission.
 Push notifications
 ==================
 
-+-----------------------+------------------------------------------------------+
-| Field                 | Description                                          |
-+=======================+======================================================+
-| Inherits from mail user                                                      |
-+-----------------------+------------------------------------------------------+
-| *No event specific fields defined*                                           |
-+---------------------+--------------------------------------------------------+
+.. versionadded:: 2.3.10
+
+   ``mailbox``, ``mail_uid``, ``failed`` and ``push_notification_event`` were added.
+
++-------------------------+------------------------------------------------------+
+| Field                   | Description                                          |
++=========================+======================================================+
+| Inherits from mail user                                                        |
++-------------------------+------------------------------------------------------+
+| mailbox                 | Mailbox for event                                    |
+| mail_uid                | Mail uid (when present)                              |
```

Comment:
"When present"? Is there more detail we should provide about when it is/isn't present?

Reply:
well, i suppose. it's there only when you have *mail* related event.

Reply:
Are there push notifications without a mail?

Reply:
yes. mailbox creation/rename/delete/subscription events.

Reply:
In that case, I'd suggest: s/when present/when the event is related to a specific mail/

Or something like that.